flake.nix

```diff
@@ -407,7 +407,7 @@
               }
               (mkIf cfg.enable {
                 environment.systemPackages = [cfg.package];
-                services.xserver.displayManager.sessionPackages = [cfg.package];
```

Comment:
Should we add a backward compatible option?

Reply:
Yes, thank you, that should be done; i overlooked that, will change that before it could be merged